test_syndiffix/oneModel.py

```diff
@@ -425,16 +425,16 @@ def oneModel(dataDir='csvGeneral',
                 # type that absharp can handle regardless of what the value are...
                 colType = 'text'
             columns.append(f"{colName}:{colTypeSymbols[colType]}")
-        abSharpArgs += " --verbose "
+        extraArgs = []
         if withFocusColumn:
-            abSharpArgs += f" --clustering-maincolumn '{focusColumn}' "
+            extraArgs = ["--clustering-maincolumn", f"'{focusColumn}'"]
```

Comment:
No need to do the wrapping in single quotes now.
```suggestion
            extraArgs = ["--clustering-maincolumn", focusColumn]
```

Reply:
So even if `focusColumn` has spaces in it, I still don't need the single quotes?

Reply:
Correct. That's because these are passed directly, without any shell pre-processing.